app/lib/features/events/pages/event_details_page.dart

```diff
@@ -0,0 +1,412 @@
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/themes/colors/color_scheme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/common/widgets/redact_content.dart';
+import 'package:acter/common/widgets/report_content.dart';
+import 'package:acter/features/events/providers/event_providers.dart';
+import 'package:acter/features/home/widgets/space_chip.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+import 'package:jiffy/jiffy.dart';
+
+class EventDetailPage extends ConsumerStatefulWidget {
+  final String calendarId;
+
+  const EventDetailPage({super.key, required this.calendarId});
+
+  @override
+  ConsumerState<EventDetailPage> createState() =>
+      _EventDetailPageConsumerState();
+}
+
+class _EventDetailPageConsumerState extends ConsumerState<EventDetailPage> {
+  @override
+  Widget build(BuildContext context) {
+    final event = ref.watch(calendarEventProvider(widget.calendarId));
+    return Scaffold(
+      backgroundColor: Theme.of(context).colorScheme.secondaryContainer,
+      body: event.when(
+        data: (calendarEvent) {
+          return CustomScrollView(
+            slivers: [
+              _buildEventAppBar(calendarEvent),
+              _buildEventBody(calendarEvent),
+            ],
+          );
+        },
+        error: (error, stackTrace) => SliverToBoxAdapter(
+          child: Text('Error loading event due to $error'),
+        ),
+        loading: () => const SliverToBoxAdapter(
+          child: Center(child: CircularProgressIndicator()),
```

Comment:
why are these `SliverToBoxAdatapter`s if they are just in the `body` and not in `slivers` of `Scaffold`? Does that even work?

Reply:
> why are these SliverToBoxAdatapters if they are just in the body and not in slivers of Scaffold?

My bad, `SliverToBoxAdapter` is by  mistake placed here. Let me remove it.

> Does that even work?   

I don't think so it will work and lead to potential error.